atd-vzd/schema/fatalities.sql

```diff
@@ -0,0 +1,128 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 14.6 (Debian 14.6-1.pgdg110+1)
+-- Dumped by pg_dump version 14.7 (Ubuntu 14.7-0ubuntu0.22.04.1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: fatalities; Type: TABLE; Schema: public; Owner: visionzero
+--
+
+CREATE TABLE public.fatalities (
+    id integer NOT NULL,
+    crash_id integer,
+    victim_name text,
+    law_enforcement_num integer,
```

Comment:
is `law_enforcement_num` unique to the individual person, or to the crash?

Reply:
this is a new field the VZ team wants added that they will be adding in the UI themselves and i dont have much insight into it yet. buuut i think its unique to a crash not a person because the place they want to eventually add functionality to add/edit is on a crash page. so now you have me thinking that this column can be removed from the fatalities table and instead needs to exist in the crashes table and the view will pull from it

Reply:
👍 yep—that sounds right to me!

Reply:
I went back to check their sharepoint, and the law_enforcement_num is taking the place of what they currently call "APD Fatal Crash" which is a number that APD assigns to crashes, like their YTD crash number. (Not YTD Fatality, so I agree crashes table sounds right)